### PR TITLE
Solution for 2016 Day 1

### PR DESCRIPTION
**Pull request by polandar**

### CODE REVIEW

**Comment by polandar** [comment]
@Kushagra-0801 pls have a look when you have time #29

**Comment by Kushagra-0801** [comment]
You input file is different than the one given by the site. Please fix that
Also submit the solution for part 2 too

**Comment by polandar** [comment]
> You input file is different than the one given by the site. Please fix that
> Also submit the solution for part 2 too

https://adventofcode.com/2016/day/1/input this is the input file, right?

**Comment by polandar** [comment]
found the missing EOL, thanks

**Comment by polandar** [comment]
@Kushagra-0801 done and done

File: 2016/Day 1 - input [hunk 1]
```diff
@@ -0,0 +1 @@
+L1, L3, L5, L3, R1, L4, L5, R1, R3, L5, R1, L3, L2, L3, R2, R2, L3, L3, R1, L2, R1, L3, L2, R4, R2, L5, R4, L5, R4, L2, R3, L2, R4, R1, L5, L4, R1, L2, R3, R1, R2, L4, R1, L2, R3, L2, L3, R5, L192, R4, L5, R4, L1, R4, L4, R2, L5, R45, L2, L5, R4, R5, L3, R5, R77, R2, R5, L5, R1, R4, L4, L4, R2, L4, L1, R191, R1, L1, L2, L2, L4, L3, R1, L3, R1, R5, R3, L1, L4, L2, L3, L1, L1, R5, L4, R1, L3, R1, L2, R1, R4, R5, L4, L2, R4, R5, L1, L2, R3, L4, R2, R2, R3, L2, L3, L5, R3, R1, L4, L3, R4, R2, R2, R2, R1, L4, R4, R1, R2, R1, L2, L2, R4, L1, L2, R3, L3, L5, L4, R4, L3, L1, L5, L3, L5, R5, L5, L4, L2, R1, L2, L4, L2, L4, L1, R4, R4, R5, R1, L4, R2, L4, L2, L4, R2, L4, L1, L2, R1, R4, R3, R2, R2, R5, L1, L2
```
Comment (Kushagra-0801) [inline 2016/Day 1 - input]: This should be
```
R4, R3, L3, L2, L1, R1, L1, R2, R3, L5, L5, R4, L4, R2, R4, L3, R3, L3, R3, R4, R2, L1, R2, L3, L2, L1, R3, R5, L1, L4, R2, L4, R3, R1, R2, L5, R2, L189, R5, L5, R52, R3, L1, R4, R5, R1, R4, L1, L3, R2, L2, L3, R4, R3, L2, L5, R4, R5, L2, R2, L1, L3, R3, L4, R4, R5, L1, L1, R3, L5, L2, R76, R2, R2, L1, L3, R189, L3, L4, L1, L3, R5, R4, L1, R1, L1, L1, R2, L4, R2, L5, L5, L5, R2, L4, L5, R4, R4, R5, L5, R3, L1, L3, L1, L1, L3, L4, R5, L3, R5, R3, R3, L5, L5, R3, R4, L3, R3, R1, R3, R2, R2, L1, R1, L3, L3, L3, L1, R2, L1, R4, R4, L1, L1, R3, R3, R4, R1, L5, L2, R2, R3, R2, L3, R4, L5, R1, R4, R5, R4, L4, R1, L3, R1, R3, L2, L3, R1, L2, R3, L3, L1, L3, R4, L4, L5, R3, R5, R4, R1, L2, R3, R5, L5, L4, L1, L1
```

File: 2016/Day 1-1.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+sides = [[-1, 0], [0, -1], [1, 0], [0, 1]]
+with open("Day 1 - input", "r") as f:
```
Comment (Kushagra-0801) [inline 2016/Day 1-1.py]: No need to specify file mode when it is only `r`

File: 2016/Day 1-2.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+sides = [[-1, 0], [0, -1], [1, 0], [0, 1]]
+with open("Day 1 - input", "r") as f:
```
Comment (Kushagra-0801) [inline 2016/Day 1-2.py]: same here

**Review by Kushagra-0801 — CHANGES_REQUESTED** [review]

File: 2016/Day 1 - input (hunk 1 above)
Comment (polandar) [reply 2016/Day 1 - input]: I believe we get different input files as we have different users but I am happy to upload yours :)

**Comment by polandar** [comment]
Other users will have different input files FYI, so if you were to get other contributors, their input files will differ from yours.

**Pull request merged**